test/index.js

```diff
@@ -1,15 +1,14 @@
 'use strict';
 
-import test from 'tape';
+import test from 'blue-tape';
 import hello from '../source/index';
 
 
-test('Tests run', (assert) => {
-  assert.pass('Tests run');
-  assert.end();
-});
+test('Tests run', (assert) => Promise.resolve(
```

Comment:
Return the resolved promise in the function body, after the assertions, both for better readability, and to keep test styles more consistent with regular tape tests (read: familiar to developers).